### DISCUSSION TITLE
How to deal with mediamtx upgrades

### DISCUSSION THREAD

**Discussion by dbuscaino** [bluenviron/mediamtx#3307] [Questions] [answered]
### Question

Hello everyone, I have a production server in which I have a running mediamtx server.
I usually find out about a new mediamtx update by  manually checking here on github. This is a bit annoying because I'm starting to have a few programs to keep update.

Do you guys know something to automate this process?

**Comment by Taps7734** [comment]
I set up a monitor on VisualPing.io to look at https://github.com/bluenviron/mediamtx/releases/
Any time there is a change to the page, I get a notification.

**Comment by aler9** [comment, marked as answer]
Hello,
Let's start from the fact that users that install the server through package managers (Homebrew, NixOS, Arch) already have this feature. Package managers provide updates.
Then there are Docker users. In Docker, updates are always manual.
People that are left out are the ones that are installing the server by using standalone binaries. We're not gonna put automatic checks in binaries due to privacy constraints. We could put something like a manual check (`mediamtx --check-updates`) and a manual upgrade (`mediamtx --upgrade`).

I've created a dedicated issue here: #3501
